charts/image-resizer/values.yaml

```diff
@@ -16,6 +16,7 @@ env: {}
 
 image:
   repository: public.ecr.aws/citizensadvice/image-resizer
+  tag: "v2.2.0"
```

Comment:
Is there a way to set this from the appVersion?

Reply:
yes, you can use the appVersion in the template as a default, experiment with a fresh chart created with `helm create` to see how it's done in the deployment template